example/CLAUDE.md

```
@@ -0,0 +1,28 @@
+# example/
+
+サンプル素材とクライアント実装を格納するディレクトリ。
+
+## ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)